src/chat/prompts.ts

```diff
@@ -0,0 +1,45 @@
+
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as Constants from "../constants/constants";
+
+export const genericChatStart = `@${Constants.mssqlChatParticipantName} Hello!`;
+
+export const analyzeQueryPerformancePrompt =
+    `@${Constants.mssqlChatParticipantName} Analyze the performance of this query. ` +
```

Comment:
Any reason why we don't use one multi line string? 
`
 Prompt Line 1
 Prompt Line 2
`

Reply:
Updated, I grabbed these prompts from PG, didn't get a chance to update them before.